lib/beaker-template.rb

```diff
@@ -1,12 +1,25 @@
+require 'stringify-hash'
+require 'beaker-template/helpers'
+require 'beaker-template/version'
+
+
 module Beaker
   module DSL
-    module Helpers
-      module Template
-        require 'stringify-hash'
-        # include your modules from the beaker-template folder. Example below:
-        require 'beaker-template/helpers'
-        require 'beaker-template/version'
-      end
+    module Template
+
+      # include your modules from the beaker-template folder. Example below:
+      include Beaker::DSL::Template::Helpers
+
     end
   end
 end
+
+
+# Boilerplate DSL inclusion mechanism:
+# First we register our module with the Beaker DSL
+Beaker::DSL.register( Beaker::DSL::Template )
```

Comment:
Was able to stub the register method using the same strategy used in beaker-hiera: that of providing a testing helper that is specifically to supply beaker methods that aren't available in testing, but will definitely be available in execution, since most beaker DSL libraries can't run stand-alone at this point.